gnomad/utils/transcript_annotation.py

```diff
@@ -177,93 +181,67 @@ def tissue_expression_ht_to_array(
 
 def preprocess_variants_for_tx(
     ht: hl.Table,
-    filter_to_genes: Optional[Union[Tuple[str], List[str]]] = None,
-    filter_to_csqs: Optional[Union[Tuple[str], List[str]]] = None,
+    filter_to_genes: Optional[List[str]] = None,
+    match_by_gene_symbol: bool = False,
+    filter_to_csqs: Optional[List[str]] = None,
     ignore_splicing: bool = True,
-    filter_to_homs: bool = False,
     filter_to_protein_coding: bool = True,
-    vep_annotation: str = "transcript_consequences",
+    vep_root: str = "vep",
 ) -> hl.Table:
     """
-    Filter variants to those that fall on transcripts of interest.
+    Prepare a Table of transcript expression information for annotation.
```

Comment:
Why is it a table of transcript expression information? I think this is a variant table without any expression information.